invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js

```diff
@@ -67,25 +33,65 @@ export default class SeriesDocuments extends Component {
     return <SeeAllButton clickHandler={goToHandler(path)} />;
   };
 
-  render() {
-    const {
-      seriesDetails,
-      showMaxDocuments,
-      seriesDocuments,
-      isLoading,
-      error,
-    } = this.props;
+  prepareData = documents => {
+    const { seriesDetails } = this.props;
+    const serials = seriesDetails.metadata.relations.serial || [];
+    const volumes = {};
+    for (const serial of serials) {
+      volumes[[serial.pid, serial.pid_type]] = serial.volume;
+    }
+    const curratedDocs = documents.hits.map(row => {
+      const key = [row.metadata.pid, 'docid'];
+      // NOTE: Not good, adding property volume to document
+      row.volume = key in volumes ? volumes[key] : '?';
+      return row;
+    });
+    return curratedDocs;
+  };
 
+  viewDetails = ({ row }) => {
+    return (
+      <Button
+        as={Link}
+        to={BackOfficeRoutes.documentDetailsFor(row.metadata.pid)}
+        compact
+        icon="info"
+        data-test={row.metadata.pid}
+      />
+    );
+  };
+
+  render() {
+    const { showMaxDocuments, seriesDocuments, isLoading, error } = this.props;
+    const curratedData = this.prepareData(seriesDocuments);
+    const columns = [
+      { title: '', field: '', formatter: this.viewDetails },
+      { title: 'ID', field: 'metadata.pid' },
+      { title: 'Volume', field: 'volume' },
+      { title: 'Title', field: 'metadata.title' },
+      {
+        title: 'Authors',
+        field: 'metadata.authors',
+        formatter: ({ col, row }) => {
+          if (col.field in row && row[col.field].length) {
```

Comment:
it might not be an array in case of errors... if you want to be super safe, probably:
```
if (_get(col.field, row) && Array.isArray(row[col.field]) {}
```